### PR TITLE
 add force overwrite to CopyFileD2 and D2Drop

### PR DESCRIPTION
**Pull request by yuvalbenshalom**
## Status
Ready
## Related Issues
fixes: https://github.com/demisto/etc/issues/15172
## Description
add the ability for file overwrite in CopyFileD2l

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests - d2script tests is not supported in content yet :(
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Scripts/script-CopyFileD2.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ script: |-
   e = e[0];
   if ( e.File ) {
       var fileName = fileNameFromEntry( args.entryid );
-      var rep = executeCommand( 'D2Drop', { destpath : args.destpath , files : fileName, using : args.system } );
+      var rep = executeCommand( 'D2Drop_copy', { destpath : args.destpath , files : fileName, using : args.system, force: args.force } );
```
Comment (anara123) [inline Scripts/script-CopyFileD2.yml]: D2Drop_copy => D2Drop

File: Scripts/script-CopyFileD2.yml [hunk 2]
```diff
@@ -33,6 +34,15 @@ args:
...
+  - "false"
+  description: Overwrite file
 scripttarget: 0
-dependson: {}
-timeout: 0s
+runonce: false
+runas: DBotWeakRole
+releaseNotes: added overwrite support to CopyFileD2
+tests:
+  - No test
```
Comment (anara123) [inline Scripts/script-CopyFileD2.yml]: please mention a reason why there are no tests

File: Scripts/script-D2Drop.yml [hunk 3]
```diff
@@ -38,4 +39,15 @@ args:
...
+  predefined:
+  - "true"
+  - "false"
+  description: Overwrite file
 scripttarget: 1
+runonce: false
+runas: DBotWeakRole
+releaseNotes: added overwrite support to D2Drop
+tests:
+  - No test
```
Comment (anara123) [inline Scripts/script-D2Drop.yml]: mention why there are no tests

**Comment by anara123** [comment]
@yuvalbenshalom BTW do you an idea how we can tests D2 scripts as part of the content build?

**Comment by anara123** [comment]
@yuvalbenshalom 
```
Validating Scripts/script-D2Drop.yml
You have failed to update id_set.json with the data of Scripts/script-D2Drop.yml please run `python Tests/scripts/update_id_set.py`
Validating Scripts/script-CopyFileD2.yml
You have failed to update id_set.json with the data of Scripts/script-CopyFileD2.yml please run `python Tests/scripts/update_id_set.py`
Exited with code 1
```
please fix so we could merge

**Comment by yuvalbenshalom** [comment]
@anara123 fixed build

**Pull request merged**